sql/src/test/java/io/crate/testing/TestingHelpers.java

```diff
@@ -76,6 +78,17 @@ public static String printedTable(Object[][] result) {
         return printRows(Arrays.asList(result));
     }
 
+    public static String printedRows(Iterable<Row> rows) {
```

Comment:
Is it possible to re-use the printedTable below somehow?  Bucket extends Iterable<Row>